lib/index.js

```diff
@@ -193,19 +194,14 @@ exports.genMigrationFile = (options, cb) => {
                         const template = Fs.readFileSync(__dirname + '/hbs/migrationFileTemplate.hbs').toString();
                         const compiler = internals.handlebars().compile(template);
                         const createdTables = tableDeltas.tableDelta.create.concat(tableDeltas.joinTableDelta.create);
-                        const sameColumn = (skipA, skipB) => skipA.tableName === skipB.tableName && skipA.column === skipB.column;
-                        const dedupedSkippedCols = skippedCols.filter((skip, i) => {
-
-                            return skippedCols.findIndex((s) => sameColumn(s, skip)) === i;
-                        });
```

Comment:
This deduping can be removed now that we prevent join table skipped columns from being re-added (See https://github.com/BigRoomStudios/schwifty-migrate-diff/pull/9/files#diff-6d186b954a58d5bb740f73d84fe39073R394)